src/cli.rs

```diff
@@ -429,6 +429,13 @@ fn subcommand_scaffold() -> Command {
             .number_of_values(1)
             .value_name("VALUE")
             .help("Semver version of the policy"),
+        Arg::new("gh-release-tag")
```

Comment:
There are no tests for the artifacthub scaffold command. However, the logic is extensively tested in PolicyEvaluator. Should we add a test here? If so, what aspects should we cover?

Reply:
I have mixed feelings about that... the only things not tested here would be the handling of the flags, reading the contents of the files (like questions.yml). I wonder if this is worth. I would track that as technical debt and deal with that later